morello/morello.yaml

```diff
@@ -0,0 +1,15 @@
+title: Morello Supplement to the Arm C Language Extensions
```

Comment:
Please rename this file to `morello_pdf_conf.yaml`. So that the name is explicit about the use we do of this file.

Please do the same for the file `mve.yaml` -> `mve_pdf_conf.yaml`